- `pip_ems_alltissues.py` was used to calculate the functionally informed PIP using EMS as a prior
- `rf_performance_comparison.py` was used to compare the performance of random forest classifier with different features or predictors (Fig. 23)
- `susie_vs_fm.py` was used to compare the functional enrichment in putative eQTLs identified in SuSiE vs FINEMAP (Fig. S4)
- `pip_ems_heatmap.py` was used to draw a heatmap that compares PIP using EMS as a prior vs uniform prior in whole blood(Fig. 2a,b, File S2, 3)
- `pip_ems_minmax100.py` was used to compare the constrained functional prior (max/min ratio < 100) (Fig. S20)
- `ems_feat_imp.py` was used to investigate the feature importance of the features in the random forest model (Fig. S7, File S1)
- `enrichments_agg.py` and `enrichments_plot.py` are used for enrichment analysis of functional features and the EMS (Fig. S3, S5, S19)
- `mpra_uwash.py` was used for MPRA saturation mutagenesis data analysis (Fig. S9a-d)
- `tissue_overview.py` was used for (Fig. S1, S2, S13)
- `enrichments_ukbb_alltissues.py` was used for quantifying the enrichment of EMS in different traits (Fig. S24)
- `locuszoom.py` was used for visualization of EMS around WSB1 gene (Fig. 2d, S15, S16)
- `basenji_effect_size.py` was used to quantify the enrichment of Basenji scores for different eQTL effect size bin (Fig. S22)
- `ems_overview.py` was used to investigate the quantify the EMS for different distance to TSS, minor allele count, and marginal association p-value bins (Fig. S21)
- `ems_qc.py` plots the feature importances and score conversions in the random forest predictor (Fig. S17, S18)
- `ems_causal_var_enrichements.py` was used to quantify the enrichment of EMS and other major scores in putative causal eQTLs and putative complex trait-causal variants (Fig. 1b, S10)
- `ems_auc_prc.py` was used to draw AUC/PRC for EMS and other major scores in prioritizing GTEx eQTLs (Fig. 1a, S8). Author: Shuvom Sadhuka. 
- `pip_ems_validation.py` was used for validation of PIP_EMS using SuRE data and UKBB fine-mapping data (Fig. 2c, S11)
- `pip_ems_and_clpp.py` was used to compare PIP_EMS and PIP_unif across tissues, and also to perform co-localization analysis (Table. 1, File S4)
